### DISCUSSION TITLE
Control RC model from PC with Multiprotocol

### DISCUSSION THREAD

**Discussion by kelvinkoko** [pascallanger/DIY-Multiprotocol-TX-Module#970] [Q&A]
I am working on project to control the RC model from PC and looking for solution that don't need modification on the RC model (receiver side)
And finally come to this project.
i am thinking to connect to a ready made multi protocol module with a USB to serial adapter. and program on PC side to send serial signal to control the vehicle. is it make sense?
or does anyone know some project already doing this that can reference to?

**Comment by pascallanger** [comment]
It has already been discussed. Search in the existing discussions.

**Comment by kelvinkoko** [comment]
ah OK, maybe this thread: https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/discussions/501
i will check about this, Thank you
